Styles.css

```diff
@@ -0,0 +1,287 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@0,300;0,400;0,700;0,900;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+/*////////////////////////
+    Table of Contents
+    1. Global Styles
+    2. Typography
+    3. Layout
+        3.1 Reusables
+        3.2 Othergit tsa
+    4. Areas
+        4.1 Header
+        4.2 Introduction
+        etc.
+    5. Media Queries
+////////////////////////*/
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+:root {
+    /*engels vertalen*/
+    --Purple: #5722CD;
+    --Blue: #4845E4;
+    --Pink: #ED4385;
+    --Black: #0C0C0C;
+    --Lightpurple: #F2F0FF;
+    --Lightgrey: #F3F6F9;
+    --Grey: #7A7A7A
+}
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Roboto', sans-serif;
+}
+
+h1, h5 {
+    font-family: 'Merriweather', serif;
+    font-size: 40px;
+}
+
+h2, h3, h4 {
+    font-family: 'Merriweather', serif;
+    font-size: 30px;
+}
+
+
+button {
+    background-color: var(--Pink);
+    font-family: 'roboto', sans-serif;
+    border-radius: 40px;
+    height: 60px;
+    width: 200px;
+    box-shadow: 0 8px 24px rgba(0, 0, 0, 0.3);
+    color: var(--Lightgrey);
+    border: none;
+    padding: 12px 24px;
+    text-transform: uppercase;
+
+}
+a:link {
+    text-decoration: none;
+}
+
+
+/* ------------------------------- 2. Typography ----------------------------------- */
+
+
+
+/* ------------------------------- 3. Layout --------------------------------------- */
+
+
+
+
+
+/* --------------- 3.1 Reusables ----------------------- */
+.outer-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    /*border: 5px solid var(--Purple);*/
+}
+
+.inner-container {
+    flex-basis: 300px;
+
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+    text-align: center;
+    margin: 40px 350px;
```

Comment:
Waarom zoveel margin aan de zijkanten? Dat is niet nodig als de inner-container gecentreerd wordt door de outer-container, toch?